CourseGrab/Models/User.swift

```diff
@@ -55,8 +55,11 @@ struct User {
 extension User {
     
     func initializeSession() {
-        guard let token = googleToken else { return }
-        NetworkManager.shared.intializeSession(googleToken: token).observe { response in
+        guard let token = googleToken else {
+            print("No token")
```

Comment:
Is this for testing? If not, let's delete it

Reply:
For now it's for testing because there are a lot of backend changes happening all the time and this helps isolate issues in the meantime.